packages/plugin-react-oxc/tsdown.config.ts

```diff
@@ -0,0 +1,14 @@
+import { defineConfig } from 'tsdown'
+
+export default defineConfig({
+  entry: 'src/index.ts',
+  define: {
+    'globalThis.__IS_BUILD__': 'true',
```

Comment:
Is define still necessary? I thought this was only for stub mode, so the branch is not necessary anymore.

https://github.com/vitejs/vite-plugin-react/blob/a5e9d52a08d747aaaedad9a45f229639f7b7238e/packages/plugin-react-oxc/src/index.ts#L16-L19